lib/plugins/aws/deploy/lib/mergeIamTemplates.js

```diff
@@ -4,6 +4,19 @@ const _ = require('lodash');
 const BbPromise = require('bluebird');
 const path = require('path');
 
+function checkStatements(statements) {
```

Comment:
Any chance to move this into an exported function and run this before `mergeIamTemplates` (which should be renamed to e.g. `mergeTemplates` or `merge`) and then export a method called `mergeIamTemplates` which is used to run everything in order? (see here: https://github.com/serverless/serverless/blob/master/lib/plugins/aws/deploy/lib/cleanupS3Bucket.js#L53-L61)

The reason for that is that the single `mergeIamTemplates` method gets more and more complex and might introduce merge conflicts in the future.